.travis.yml

```diff
@@ -1,8 +1,10 @@
 language: php
+sudo: false
 php:
   - "5.4"
-install:
+before_install:
   - composer self-update

```

Comment:
@GrahamCampbell Please take a look at the pull request for the reasoning behind that.